channels/cz.m3u

```diff
@@ -1,82 +1,48 @@
-https://nova-live.ssl.cdn.cra.cz/channels/nova_action_avod/playlist/cze/live_xq.m3u8
-#EXTINF:-1 tvg-id="NovaAction.cz" tvg-country="CZ" tvg-language="Czech" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/d/d2/Nova-action-logo-2017.png" group-title="",Nova Action (576p) [Offline]
-https://nova-live.ssl.cdn.cra.cz/channels/nova_action_avod/playlist/cze/live_hq.m3u8
-#EXTINF:-1 tvg-id="NovaCinema.cz" tvg-country="CZ" tvg-language="Czech" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/c/cd/Nova_Cinema_logo.png" group-title="",Nova Cinema (640p) [Offline]
-https://nova-live.ssl.cdn.cra.cz/channels/nova_cinema_avod/playlist/cze/live_xq.m3u8
-#EXTINF:-1 tvg-id="NovaCinema.cz" tvg-country="CZ" tvg-language="Czech" tvg-logo="https://static.epg.best/cz/NovaCinema.cz.png" group-title="",Nova Cinema (576p) [Offline]
-https://nova-live.ssl.cdn.cra.cz/channels/nova_cinema_avod/playlist/cze/live_hq.m3u8
-#EXTINF:-1 tvg-id="NovaGold.cz" tvg-country="CZ" tvg-language="Czech" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/7/7f/Nova_Gold_logo.png" group-title="",Nova Gold (640p) [Offline]
-https://nova-live.ssl.cdn.cra.cz/channels/nova_gold_avod/playlist/cze/live_xq.m3u8
-#EXTINF:-1 tvg-id="NovaGold.cz" tvg-country="CZ" tvg-language="Czech" tvg-logo="https://static.epg.best/cz/NovaGold.cz.png" group-title="",Nova Gold (576p) [Offline]
-https://nova-live.ssl.cdn.cra.cz/channels/nova_gold_avod/playlist/cze/live_hq.m3u8
-#EXTINF:-1 tvg-id="Ocko.cz" tvg-country="CZ;SK" tvg-language="Czech" tvg-logo="" group-title="Music",Óčko (540p)
-https://ocko-live.ssl.cdn.cra.cz/channels/ocko/playlist.m3u8
-#EXTINF:-1 tvg-id="Ocko.cz" tvg-country="CZ;SK" tvg-language="Czech" tvg-logo="" group-title="Music",Óčko (540p)
-https://ocko-live.ssl.cdn.cra.cz/channels/ocko/playlist/cze/live_hq.m3u8
-#EXTINF:-1 tvg-id="OckoExpres.cz" tvg-country="CZ" tvg-language="Czech" tvg-logo="" group-title="Music",Óčko Expres (540p)
-https://ocko-live.ssl.cdn.cra.cz/channels/ocko_expres/playlist/cze/live_hq.m3u8
-#EXTINF:-1 tvg-id="OCKOEXPRESmusic.cz" tvg-country="CZ" tvg-language="" tvg-logo="" group-title="Music",OCKO EXPRES Music (360p)
-http://ocko-live.ssl.cdn.cra.cz/channels/ocko_expres/playlist/cze/live_lq.m3u8?offsetSeconds=0
-#EXTINF:-1 tvg-id="OckoGold.cz" tvg-country="CZ" tvg-language="Czech" tvg-logo="https://i.imgur.com/nrWNyLc.jpg" group-title="Music",Óčko Gold (540p)
-https://ocko-live-dash.ssl.cdn.cra.cz/cra_live2/ocko_gold.stream.1.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="OckoStar.cz" tvg-country="CZ" tvg-language="Czech" tvg-logo="" group-title="Music",Óčko Star (540p)
-https://ocko-live.ssl.cdn.cra.cz/channels/ocko_gold/playlist/cze/live_hq.m3u8
-#EXTINF:-1 tvg-id="OckoTV.cz" tvg-country="CZ" tvg-language="" tvg-logo="https://i.imgur.com/jKPRYdM.png" group-title="Music",Ocko TV (540p)
-https://ocko-live-dash.ssl.cdn.cra.cz/cra_live2/ocko.stream.1.smil/playlist.m3u8
```

Comment:
![image](https://user-images.githubusercontent.com/5430116/134543324-24112bf6-4142-40bb-965b-a0e2c3466376.png)

All Ocko Links do work for me. Are they duplicating something?